apix-integrationtests/alfresco/src/main/java/eu/xenit/apix/tests/search/SearchServiceFacetsTestJavaApi.java

```diff
@@ -85,7 +86,8 @@ public void TestGetWithFacetsIncludesCustomFilterFacets() throws InterruptedExce
             }
             assertTrue("'" + facetQName + "' not found in returned facets.", isFacetFound);
         } finally {
```

Comment:
Here the test breaks due to code below. Deleted and will give no issues for our CI